File: 1238.py
x = int(input())
for k in range(x):
    i=0
    j=0
    str = input().split()
    str1 = str[0]
    str2 = str[1]
    len1 = int(len(str1))
    len2 = int(len(str2))
    while(1):
        if(i<len1):
            print(str1[i], end="")
            i+=1
        if(j<len2):
            print(str2[j], end="")
            j+=1
        if(i==len1 and j==len2):
            break
    print("")

File: 1238.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>

int main(int argc, char const *argv[]) {
  int k,x;
  char *str1,*str2;
  scanf("%d",&x);
  for(k=0;k<x;k++){
  int i=0,j=0,len1,len2;
  str2 = (char*)malloc(sizeof(char));
  str1 = (char*)malloc(sizeof(char));
  scanf("%s",str1);
  scanf("%s",str2);
  len1 = strlen(str1);
  len2 = strlen(str2);
  while(1){
    if(i<len1){
      printf("%c",str1[i]);
      i++;}
    if(j<len2){
      printf("%c",str2[j]);
      j++;}
    if((i == len1) && (j == len2))
        break;
  }
  printf("\n");}
  return 0;
}
